spec/forms/registration/terms_and_conditions_form_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Registration::TermsAndConditionsForm do
+  subject(:form) { described_class.new(user: user) }
+
+  let(:user) { create(:user) }
+
+  describe '#validate' do
+    let(:terms_and_conditions_errors) { form.errors[:terms_and_conditions_agreed_at] }
```

Comment:
Love the way you have structured the spec, shorten this variable to `errors` and it'll read even better IMHO

Reply:
Also is the GovOne only?